package-lock.json

```diff
@@ -447,442 +530,36 @@
     "balanced-match": {
       "version": "1.0.0",
       "resolved": "https://registry.npmjs.org/balanced-match/-/balanced-match-1.0.0.tgz",
-      "integrity": "sha1-ibTRmasr7kneFk6gK4nORi1xt2c=",
-      "dev": true
+      "integrity": "sha1-ibTRmasr7kneFk6gK4nORi1xt2c="
     },
     "batch-processor": {
       "version": "1.0.0",
       "resolved": "https://registry.npmjs.org/batch-processor/-/batch-processor-1.0.0.tgz",
       "integrity": "sha1-dclcMrdI4IUNEMKxaPa9vpiRrOg="
     },
     "bcrypt": {
-      "version": "3.0.2",
-      "resolved": "https://registry.npmjs.org/bcrypt/-/bcrypt-3.0.2.tgz",
-      "integrity": "sha512-kE1IaaRchCgdrmzQX/eBQKcsuL4jRHZ+O11sMvEUrI/HgFTQYAGvxlj9z7kb3zfFuwljQ5y8/NrbnXtgx5oJLg==",
+      "version": "1.0.3",
```

Comment:
I'm a bit weirded out by the bcrypt version downgrade here; any ideas what's going on?

Reply:
The downgrade happened in 2cbfd20b153fbc12e0373baab13ffa61d5c8643a; I think it just didn't get picked up by the lock file then for some reason.